src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -33,9 +34,17 @@ public function indexAction()
     {
         $this->disableInProduction();
 
+        $summary = $this->get(SummaryManager::class)->getForAdherent($this->getUser());
+        $pathImage = 'images/'.$summary->getMemberUuid().'.jpg';
```

Comment:
What if the summary does not exist yet? Maybe we should use the summary slug instead of the adherent uuid, or both?